<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>results_screen</class>
 <widget class="QMainWindow" name="results_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trails</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Application by Brendan Cook</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QLabel" name="trail_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>491</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trail_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>491</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trail_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>491</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trail_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>491</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trail_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>491</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="trail_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>581</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trail Name | Difficulty | Miles | Elevation Gain</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
